.agents/skills/wpilog-parser/README.md

```
@@ -0,0 +1 @@
+Skill for https://github.com/jonahsnider/wpilog-parser/
```

markdownlint-cli2 (0.23.1)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)